### PR TITLE
update aicoe-ci config to build container image using repo containerfile

### PR DESCRIPTION
**Pull request by 4n4nd**
Resolves https://github.com/operate-first/hitchhikers-guide/issues/120

### CODE REVIEW

**Comment by 4n4nd** [comment]
I think the Containerfile is building a base image that we could use to build tha hitchhiker's guide image.
@tumido is that the case here?

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1,12 +1,24 @@
 check:
-  - thoth-build
+  - overlays-build
+overlays_dir: "."
+overlays:
+  - name: builder-image
```
Comment (harshad16) [inline .aicoe-ci.yaml:5]: As we are using the overlays method, and it is described in this PR
that the `overlays_dir` is `.`

then the pipeline would try to look for the sub_dir with name `builder-image` at the root
for the containerfile


and same fo the hitchhikers-guide.

**Review by harshad16 — CHANGES_REQUESTED** [review]
need some more further changes

**Comment by 4n4nd** [comment]
Looks like we don't need this PR anymore, the builder image is directly being built in quay and the htg image should build

**Pull request closed without merging**